src/unix/linux_like/linux/arch/generic/mod.rs

```diff
@@ -257,6 +257,7 @@ cfg_if! {
         pub const RLIMIT_NICE: ::__rlimit_resource_t = 13;
         pub const RLIMIT_RTPRIO: ::__rlimit_resource_t = 14;
         pub const RLIMIT_RTTIME: ::__rlimit_resource_t = 15;
+        #[allow(deprecated)]
         pub const RLIMIT_NLIMITS: ::__rlimit_resource_t = RLIM_NLIMITS;
```

Comment:
Hmm.  Come to think of it, we'll probably have to mark `RLIMIT_NLIMITS` as deprecated, too.